common/app/views/fragments/javaScriptLaterSteps.scala.html

```diff
@@ -335,6 +323,118 @@
                 });
 
             });
+    </script>
+}
+
+@bootSystemJs() = {
+
+    <script>
+        @Html(Static.js.es6ModuleLoader)
+        @Html(Static.js.systemJs)
+        @Html(Static.js.systemJsAppConfig)
+
+        @if(!play.Play.isDev()) {
+            @Html(Static.js.systemJsBundleConfig)
+        }
+
+        System.paths['ophan/ng']  = '@{Configuration.javascript.config("ophanJsUrl")}.js';
+        System.paths['googletag'] = '@{Configuration.javascript.config("googletagJsUrl")}';
+
+        @Html(Static.js.systemJsNormalize)
+
+        window.require = System.amdRequire.bind(System);
+        window.define = System.amdDefine.bind(System);
+
+        Promise.all(
```

Comment:
What is this magic `Promise` object you speak of?? We don't have they polyfill loaded for this at this point and therefore will throw on older devices...